docs/nitclk/class.rst

```diff
@@ -1,442 +1,445 @@
-.. py:module:: nitclk
-
-Session
-=======
-
-.. py:class:: Session(self)
-
-    
-
-    TBD
 
-    
+.. py:module:: nitclk
 
+Public API
+==========
 
+NI-TClk python module consists of several functions and properties. The functions are not sessions based
+like other nimi-python driver support and are called directly on the nitclk module. See :py:class:`SessionReference`
+for properties.
```

Comment:
* Capitalize Python.
* I don't think you need to contrast with the rest of the drivers... describe for what it is, not for what it isn't.
* "are called directly on the nitclk module" seems redundant for a Python developer.

Here's an attempt:

```
The `nitclk` module provides synchronization facilites to allow multiple instruments to simultaneously respond to triggers, to align Sample Clocks on multiple instruments, and/or to simultaneously start multiple instruments.
It consists of a set of functions that act on a list of nitclk.SessionReference objects. nitclk.SessionReference also has a set of properties for configuration.
```

Reply:
Changed, but also referenced Session classes:
```
The `nitclk` module provides synchronization facilites to allow multiple instruments to simultaneously
respond to triggers, to align Sample Clocks on multiple instruments, and/or to simultaneously start multiple
instruments.

It consists of a set of functions that act on a list of :py:class:`SessionReference` objects or nimi-python `Session`
objects for drivers that support NI-TClk. :py:class:`SessionReference` also has a set of properties for configuration.
```